app/concepts/user/create.rb

```diff
@@ -0,0 +1,12 @@
+class User::Create < Trailblazer::Operation
+  include Model
+  model User
+
+  def process(params)
+    @model = User.new
```

Comment:
Agreed with KL. Also, since every operation contains a form object then this is the place where we could validate the object before we persist it in the database.